CHANGELOG.next.asciidoc

```diff
@@ -176,6 +176,13 @@ https://github.com/elastic/beats/compare/v6.6.0...6.x[Check the HEAD diff]
 - Release Couchbase module as GA. {pull}10201[10201]
 - Release aerospike module as GA. {pull}10203[10203]
 - Making RabbitMQ Metricbeat module GA. {pull}10165[10165]
+- Release mongodb.metrics and mongodb.replstatus as GA. {pull}10242[10242]
+- Release mysql.galera_status as GA. {pull}10242[10242]
```

Comment:
I think `galera_status` was fine as beta/experimental, it doesn't have any test and we may merge it into `status` at some moment.

Reply:
@jsoriano do you prefer to keep it experimental in case we can merge it or better beta?

Reply:
BTW it also means we need to revert it in master #10242

Reply:
We decided to go with beta. I opened https://github.com/elastic/beats/pull/10283 and will adjust it in this PR directly.